src/components/shift-schedule/ScheduleCellContent.tsx

```diff
@@ -0,0 +1,70 @@
+import type { ScheduleCellProps } from '@/components/shift-schedule/ScheduleTable';
+import { TableCell } from '@/components/ui/Table';
+import { cx } from '@/lib/utils';
+import { UserIcon, UsersIcon } from 'lucide-react';
+import { useTranslations } from 'next-intl';
+import type React from 'react';
+
+function ScheduleCellContent({ members }: ScheduleCellProps) {
+  const t = useTranslations('shiftSchedule.scheduleTable.scheduleCellContent');
+
+  const colorStyle =
+    members.length === 0
+      ? 'bg-accent/50 hover:bg-accent dark:bg-accent/40 dark:hover:bg-accent/60 text-accent-foreground'
+      : 'bg-foreground/20 hover:bg-foreground/25';
+
+  let memberCountIcon: React.ReactNode;
+  const memberCountIconStyle = 'w-7 h-7';
+  let memberCount: React.ReactNode;
+  const memberCountStyle = 'flex align-bottom space-x-1 space-y-0';
+  let skillIcons: React.ReactNode;
+
+  // Set member count icon, member count, and skill icons based on amount of members present
+  if (members.length === 0) {
+    // Empty shift
+    memberCount = <p className='leading-none'>{t('empty')}</p>;
+  } else {
+    // At least 1 person on shift
+    skillIcons = (
+      <section className='leading-none'>[skill icons total]</section>
+    );
+
+    if (members.length === 1) {
```

Comment:
The part after `if (members.length === 1)` can also be simplified by moving it out of the else part of the last if statement